README.md

```diff
@@ -46,6 +49,20 @@
 9. have a way for users to contact the folks operating the site (email, Discord, etc.).
 10. notify users beforehand when your TOS or privacy policy is changing.
 11. not be a copy of other's bot lists (open sourced lists, including lists that are edited or modified from other's lists).
+12. have a better way of handling errors, instead of showing console errors, show a custom 500 status code page.
+13. load fast enough, and use not a lot of time and/or internet data to load your site for the first time.
+14. have a caching system in place.
+15. not have a custom keybind to stop browser dev tools from loading.
+16. save users from getting xss attacks.
+17. sanitize bot's long description.
+18. allow custom css.
+19. store markdown, render it as html on user's request.
+20. be usable on safari web browsers (both on iOS and Mac), Apple breaks your site on safari most of time, please check it out (maybe via browser stack).
+21. not have NSFW ads, whilst following rule no. 7
+22. not share user sensitive data without user's consent.
```

Comment:
```suggestion
```
No point listing the law here, and definitely not under _should_

Reply:
A reminder that this section is for good bot lists, so having a should doesn't matters for lists that are not good...?